mobile/lib/screens/manage-vehicles/select_vehicle_type_screen.dart

```diff
@@ -8,8 +8,9 @@ import 'package:mobile/utils/widget_functions.dart';
 import 'package:carousel_slider/carousel_slider.dart';
 
 import '../../colors.dart';
+import 'add_vehicle_screen.dart';
 
-var reports = [
+var selectVehicle = [
   {'title': 'assets/images/car.png', 'content': 'Car'},
```

Comment:
Can we use the same VehicleType enum I had defined within 'models' folder here?

So that you can replace `'content': 'Car' ` with `'content': '${VehicleType.car.name.toUpperCase()}'`